Library Conflict with libfbjni.so

### Description

I encountered an issue while building my React Native project (version 0.76.0) after installing the @react-navigation/native package.
After installing react-native-screens, as per @react-navigation/native documentation, when I attempt to build the project with "Make Project", I receive the following error:

Execution failed for task ':react-native-screens:mergeDebugAndroidTestNativeLibs'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.MergeNativeLibsTask$MergeNativeLibsTaskWorkAction
   > 2 files found with path 'lib/arm64-v8a/libfbjni.so' from inputs:
      - /Users/name_user/Desktop/GitHub/test-farmer-app/testfarmer/node_modules/react-native-screens/android/build/intermediates/library_jni/debug/copyDebugJniLibsProjectOnly/jni/arm64-v8a/libfbjni.so
      - /Users/name_user/.gradle/caches/8.10.2/transforms/2fa3c78e855de2c4ef8aa6911c4216a0/transformed/react-android-0.76.0-debug/jni/arm64-v8a/libfbjni.so
     If you are using jniLibs and CMake IMPORTED targets, see
     https://developer.android.com/r/tools/jniLibs-vs-imported-targets

I tried excluding on "android/app/build.gradle" with

android {
    packagingOptions {
        // Escludi le librerie specifiche nella cache di Gradle
        exclude "/Users/giannigulino/.gradle/caches/**/lib/arm64-v8a/libfbjni.so"
        exclude "/Users/giannigulino/.gradle/caches/**/lib/x86_64/libfbjni.so"
    }
} 

But I didn't solve it.


### Steps to reproduce

1. Install React Native 0.76.0
2. Install with "yarn add @react-navigation/native"
3. Install with "yarn add react-native-screens react-native-safe-area-context"
4. From Android Studio, "Make Project"

### Snack or a link to a repository

./

### Screens version

3.35.0

### React Native version

0.76.0

### Platforms

Android

### JavaScript runtime

None

### Workflow

None

### Architecture

None

### Build type

None

### Device

None

### Device model

_No response_

### Acknowledgements

Yes

Hi @kkafar, it seems that clearing the cache/build has made some progress:

> I would start with removing all the caches, including gradle: ~/.gradle/caches, node_modules, android/app/build android/app/.cxx android/build.

Now, i obtain this error:

FAILURE: Build failed with an exception.

* What went wrong:
Cannot locate tasks that match ':react-native-screens:testClasses' as task 'testClasses' not found in project ':react-native-screens'.

P.S: In addition to the above attempts, I have updated the version of react-native-screens to 4.3.0.

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -159,7 +159,13 @@ android {
                 "**/libc++_shared.so",
                 "**/libreact_render*.so",
                 "**/libreactnativejni.so",
-                "**/libreact_performance_timeline.so"
+                "**/libreact_performance_timeline.so",
+                // In 0.76 multiple react-native's libraries were merged and these are the main new artifacts we're using.
+                // Some of above lib* names could be removed after we remove support for 0.76.
+                // https://github.com/facebook/react-native/pull/43909
+                // https://github.com/facebook/react-native/pull/46059
+                "**/libfbjni.so", 
+                "**/libreactnative.so"
         ]
     }
     sourceSets.main {

